pySDC/implementations/problem_classes/Battery.py

```diff
@@ -280,11 +270,131 @@ def solve_system(self, rhs, factor, u0, t):
 
         return me
 
+
+class battery_n_condensators(ptype):
```

Comment:
What is the difference between this class and the `battery` class? Both take `ncondensators` as essential keys.

Reply:
Hm. Battery has only one capacitor. And this one has any number of capacitors. I am asking myself why I used the german word and put in an englisch writing sense named condensators instead of writing capacitors.. Facepalming.. Anyway. Maybe I can say that battery_n_condensators inherits from battery, but I need to add almost each method instead of count_switches()